Src/WitsmlExplorer.Frontend/components/ContentViews/RigsListView.tsx

```diff
@@ -13,15 +13,43 @@ export const RigsListView = (): React.ReactElement => {
       setRigs(selectedWellbore.rigs);
     }
   }, []);
+  const getTableData = () => {
+    return rigs.map((rig) => {
+      return {
+        ...rig,
+        id: rig.uid,
+        ratingDrillDepth: `${rig.ratingDrillDepth?.value ?? ""} ${rig.ratingDrillDepth?.uom ?? ""}`,
+        ratingWaterDepth: `${rig.ratingWaterDepth?.value ?? ""} ${rig.ratingWaterDepth?.uom ?? ""}`,
+        airGap: `${rig.airGap?.value ?? ""} ${rig.airGap?.uom ?? ""}`,
+        isOffshore: `${rig.isOffshore.toString()}`,
```

Comment:
isOffshore is defined as string so it is redundant to call toString()

Reply:
isOffshore is a bool so it has to be parsed, what do you think?